integration-test/src/main/java/com/scalar/db/api/TwoPhaseCommitTransactionIntegrationTestBase.java

```diff
@@ -55,37 +56,50 @@ public abstract class TwoPhaseCommitTransactionIntegrationTestBase {
           .addClusteringKey(ACCOUNT_TYPE)
           .addSecondaryIndex(SOME_COLUMN)
           .build();
-  private DistributedTransactionAdmin admin;
-  private TwoPhaseCommitTransactionManager manager;
+  private DistributedTransactionAdmin admin1;
+  private DistributedTransactionAdmin admin2;
+  private TwoPhaseCommitTransactionManager manager1;
+  private TwoPhaseCommitTransactionManager manager2;
```

Comment:
Separated a `TwoPhaseCommitTransactionManager` instance per table because we assume users execute 2PC transactions with different instances. It's more like an actual situation.